docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -122,7 +131,7 @@ In any target block, you can use the following declarations:
 |---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
 | `attributes`        | Attributes used for [disambiguating targets](multiplatform-set-up-targets.md#distinguish-several-targets-for-one-platform) for a single platform. |
 | `preset`            | The preset that the target has been created from, if any.                                                                                         |
-| `platformType`      | Designates the Kotlin platform of this target. Available values: `jvm`, `androidJvm`, `js`, `native`, `common`.                                   |
+| `platformType`      | Designates the Kotlin platform of this target. Available values: `jvm`, `androidJvm`, `js`, `native`, `common`, `wasmJs`, `wasmWasi`.             |
```

Comment:
`platformType` are the same for `wasmJs` and `wasmWasi`, just `wasm`, this field is about "which compiler should be used"
https://github.com/JetBrains/kotlin/blob/0e861e386e4b252d4052cd214a9bcefc95249b2d/libraries/tools/kotlin-gradle-plugin-api/src/common/kotlin/org/jetbrains/kotlin/gradle/plugin/KotlinPlatformType.kt#L17